code/script.js

```diff
@@ -0,0 +1,56 @@
+var flag = false;
+
+// displays numerous inputs (numbers) on the screen
+function displaynumber(n1) {
+  calculator.display.value = calculator.display.value + n1;
+}
+
+// testing 0 input, when 0 comes after / buttons are disabled and message is given
+function testZero(n2) {
+  if (flag == true && n2 == 0) {
+    var b = document.getElementsByClassName("btn");
+    for (var i = 0; i < b.length; i++) {
+      b[i].disabled = true;
+      b[i].style.color = "black";
+      document.getElementById("disp2").value = "Can not devide by 0! Click C";
+    }
+  } else {
+    flag = false;
+  }
+}
+
+// sets warning, division is in play, boolean used
+function setWarning() {
+  flag = true;
+}
+
+// clears both displays, resets boolean, enables buttons
+function cl() {
+  flag = false;
+  document.getElementById("disp1").value = "";
```

Comment:
Please, dont use shortcuts when it is not needed because of too long words. Also try to change number from word. 
displayTop / displayMain / displaySample  ... same for bottom display .... displayLog / displayInfo would be more better

Reply:
top display name is now name="nameDisplayMain"
bottom one is called name="nameDisplayInfo"